### PR TITLE
D training

### PR DESCRIPTION
**Pull request by dpatel77**

### CODE REVIEW

**Review by prashantkul — COMMENTED** [review]
@dpatel77 maybe trainer.py could be empty for now and when @SVeldman develops the model, we can build this class?

**Comment by dpatel77** [comment]
Yes, that was a mistake. Thanks for catching it. Will fix in the next push.

File: vision_project/vision_models/submission_notebook.ipynb
Comment (prashantkul) [inline vision_project/vision_models/submission_notebook.ipynb:1]: @dpatel77 Can we move this notebook to one level up, in the "vision_project" folder?

Comment (dpatel77) [reply vision_project/vision_models/submission_notebook.ipynb:1]: Yup, will do it on the next push

File: README.md [hunk 1]
```diff
@@ -47,3 +47,31 @@ There are overall 75 labels for this dataset. 25 main labels and 3 secondary lab
   * Normal/Mild 
   * Moderate 
   * Severe
+
+# Format of Github Repo
```
Comment (prashantkul) [inline README.md:51]: We should update this too. 

vision_models = core code for the models and related classes

**Pull request closed without merging**